.yardopts

```diff
@@ -1,7 +0,0 @@
---no-private
```

Comment:
what is reason to drop yardopts?

Reply:
I do not know anything about it.

Reply:
IMHO yard will be not used in this package at all.